axelrod/tournament.py

```diff
@@ -376,4 +376,4 @@ def __init__(self, players, edges, match_generator=SpatialMatches,
 
         self.edges = edges
         self.match_generator = SpatialMatches(
-            players, turns, self.game, repetitions, edges)
+            players, turns, self.game, repetitions, self.noise, edges)
```

Comment:
Yeah add another one, that just tests that noise has been passed to the tournament, not that the tournament has passed the noise to it's generator. Which is what the bug is here.

So have:

```
self.assertEqual(tournament.noise, tournament.match_generator.noise)
```